demos/render/lv_demo_render.c

```diff
@@ -768,6 +768,21 @@ static lv_obj_t * create_blend_mode_obj(lv_obj_t * parent, int32_t col, int32_t
     return obj;
 }
 
+static void canvas_draw_buf_reshape(lv_draw_buf_t * draw_buf)
+{
+#if LV_USE_DRAW_VG_LITE
+    /* VG-Lite requires automatic stride calculation */
+    lv_draw_buf_t * buf = lv_draw_buf_reshape(draw_buf,
+                                              draw_buf->header.cf,
+                                              draw_buf->header.w,
+                                              draw_buf->header.h,
+                                              LV_STRIDE_AUTO);
+    LV_ASSERT_MSG(buf == draw_buf, "Reshape failed");
```

Comment:
```suggestion
    lv_result_t res = lv_draw_buf_adjust_stride(draw_buf, LV_STRIDE_AUTO);
    LV_ASSERT_MSG(res == LV_RESULT_OK, "Reshape failed");
```

Reply:
`lv_draw_buf_adjust_stride` will have an additional `lv_memmove` operation, `lv_draw_buf_reshap`e will only recalculate the stride and estimate the remaining space, without any additional operations.

Reply:
Fair.  Test sometimes act like example, let's follow the optimized way.